README.md

```diff
@@ -12,55 +12,101 @@ poetry add feature-store-bundle
 
 # Getting started
 
+1. Define entities in `config.yaml` of your Daipe project
```

Comment:
I would probably just delete the README contents completely, there is no way for us to keep it up to date at all times

Reply:
and nobody ever reads it :)

Reply:
How about you @guderkar ? :)